mobile_bkend/mobile_setup_simple.sh

```diff
@@ -0,0 +1,9 @@
+apt update
+
+apt install -y openssh-server proot-distro termux-api p7zip python-pip
+
+termux-setup-storage
+
+yes | pip install gdown
+
+gdown https://drive.google.com/file/d/1-v4MsH36HbbTdzMX1RpQ2-avj8W9clwy/view?usp=sharing
```

Comment:
What's this random file being downloaded?

Reply:
This is the container that already has FedScale and its dependencies installed.